app/controllers/pets_controller.rb

```diff
@@ -31,6 +31,7 @@ def update
   end
 
   def destroy
+    favorites.remove_pet_id(params[:pet_id])
```

Comment:
Using our remove_pet_id method we created to remove the pet's id from favorites when we delete and pet.